### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#326] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 4.1**: Sia $V$ uno spazio vettoriale. Dimostra che $0_V=O$ per ogni $v\in V$ utilizzando solamente le altre proprietà della definizione di uno spazio vettoriale.
> Suggerimento: parti da $0+0=0$

**Esercizio 4.2**: Sia $V$ uno spazio vettoriale. Dimostra che $(-1)v+v=O$ e che $(-1)v=-v$ per ogni $v\in V$. Come conseguenza, dimostra che se $v,v_1,v_2\in V$ sono tali che $v_1+v=O=v_2+v$, allora $v_1=v_2$ ; in altre parole l'opposto di un vettore è univocamente determinato.

**Esercizio 4.3**: Sia $V$ uno spazio vettoriale sul campo $\mathbb{K}$, e prendiamo $v\in V$ e $\lambda \in \mathbb{K}$. Dimostra che $\lambda v=O$ se e solo se $\lambda =0$ oppure $v=O$

**Comment by daromacarlo** [comment]
![Immagine 2023-11-22 113020](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/1990b325-21b6-4cf6-8636-32c2080dd6ac)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-22 113109](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/38b8fd3c-ec34-4278-a4b3-e1fcb24d533d)

**Comment by IonutCicio** [comment]
@CarloDaRomadev Il punto 4.1 è sbagliato, perché $0 \in \mathbb{R} \neq \underline{0} \in V$, quando scriviamo $0\underline{v} = \underline{0}$, lo $0$ a sinistra appartiene a $\mathbb{R}$, quello a destra appartiene a $V$, quindi non puoi usare direttamente gli assiomi in questo modo _(suppongo)._

Mettiamo per assurdo che $0_\mathbb{R} \underline{v} = \underline{w} \neq \underline{0}$, possiamo scrivere $\underline{v} = 1_{\mathbb{R}} \cdot \underline{v}$ per gli assiomi, quindi 

$$
\underline{v} = 1_{\mathbb{R}} \underline{v} = (0_\mathbb{R} + 1_\mathbb{R})  \underline{v} = 0_\mathbb{R}\underline{v} + 1_\mathbb{R} \underline{v} = \underline{w} + \underline{v}
$$ 

- $1_\mathbb{R} \underline{v} = \underline{v}$ per gli assiomi
- $1_\mathbb{R} = 0_\mathbb{R} + 1_\mathbb{R}$ per come è definita la somma in $\mathbb{R}$ 
- Infine si applica l'assioma della distributività

Ora si ha che $\underline{v} = \underline{w} + \underline{v}$, ma, sapendo che $(V, +)$ è un gruppo commutativo, sappiamo che $\forall \underline{v} \in V, \exists \underline{0} : \underline{v} + \underline{0} = \underline{v} = \underline{0} + \underline{v}$ e si dimostra che questo $\underline{0}$, elemento neutro, è unico, quindi abbiamo un assurdo, dato che $\underline{w}$ deve essere per forza $\underline{0}$

#### Il punto 4.2 non è propriamente sbagliato, lo farei diversamente

Supponiamo per assurdo che $(-1)_\mathbb{R} \underline{v} = \underline{w} \neq -\underline{v}$. Sappiamo che 

$$
\underline{0} = 0_\mathbb{R} \underline{v} = (1_\mathbb{R} - 1_\mathbb{R}) \underline{v} = 1_\mathbb{R} \underline{v} + (-1)_\mathbb{R} \underline{v} = \underline{v} + \underline{w}
$$

Quindi abbiamo $\underline{0} = \underline{v} + \underline{w}$, ma, essendo $(V, +)$ un gruppo commutatitvo, per cui $\forall \underline{v} \in V, \exists -\underline{v} \in V : \underline{v} + (-\underline{v}) = \underline{0} = -\underline{v} + \underline{v}$, si dimostra che questo elemento $-\underline{v}$ è unico, quindi abbiamo un assurdo, dato che $\underline{w}$ deve essere per forza $-\underline{v}$

_(il terzo punto non l'ho controllato, e non sono sicuro che queste si possano chiamare propriamente "dimostrazioni per assurdo")_

**Comment by daromacarlo** [comment]
@CuriousCI grazie

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c904f159-c544-42e1-a771-a3b5ac7e3f40)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![Screenshot 2023-11-29 112204](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f2d3b27a-941a-427f-b419-e1741b490d02)

![Screenshot 2023-11-29 112227](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/119aeb97-deb0-456f-af51-57cb7079a47d)

![Screenshot 2023-11-29 112308](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b8205c66-a997-414a-ae69-3ee82490d839)
![Screenshot 2023-11-29 112320](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/580c4bf4-ec39-4178-9385-df253d2f8465)
